src/platform/forms/tests/save-in-progress/01-sip-review.cypress.spec.js

```diff
@@ -21,7 +21,7 @@ describe('SIP Review Test', () => {
     cy.login(mockUser);
 
     cy.visit(reviewUrl);
-    cy.get('body').should('be.visible');
+    cy.get('bodddy').should('be.visible');
```

Comment:
Is this a typo?

Reply:
This is a deliberate break for testing purposes.

Reply:
We were forcing some failures to test a feature.